Detect ninja executable alternate name (again)

Trying to build latest version of tcnative, fails on system having ninja-build (similar to issue #475).

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:1.8:run (build-libressl-non-windows) on project netty-tcnative-libressl-static: An Ant BuildException has occured: Execute failed: java.io.IOException: Cannot run program "ninja" (in directory "/u01/netty/netty-tcnative/libressl-static/target/libressl-3.0.2/build-ninja"): error=2, No such file or directory
[ERROR] around Ant part ...<exec resolveexecutable="true" failonerror="true" dir="/u01/netty/netty-tcnative/libressl-static/target/libressl-3.0.2/build-ninja" executable="ninja"/>... @ 21:167 in /u01/netty/netty-tcnative/libressl-static/target/antrun/build-main.xml
```

--- a/libressl-static/pom.xml
+++ b/libressl-static/pom.xml
@@ -261,7 +261,17 @@
                           <arg value="-GNinja" />
                           <arg value=".." />
                         </exec>
-                        <exec executable="ninja" failonerror="true" dir="${libresslBuildDir}" resolveexecutable="true" />
+                        <if>
+                          <!-- may be called ninja-build or ninja -->
+                          <!-- See https://github.com/netty/netty-tcnative/issues/475 -->
+                          <available file="ninja-build" filepath="${env.PATH}" />
+                          <then>
+                            <exec executable="ninja-build" failonerror="true" dir="${libresslBuildDir}" resolveexecutable="true" />
+                          </then>
+                          <else>
+                            <exec executable="ninja" failonerror="true" dir="${libresslBuildDir}" resolveexecutable="true" />
+                          </else>
+                        </if>
                       </else>
                     </if>
                   </target>

